compiler/testData/codegen/boxInline/simple/inlineCallInInlineLambda.kt

```diff
@@ -0,0 +1,8 @@
+inline fun f(g: (Int) -> Unit) = g(0)
```

Comment:
Please split test into two FILE sections (see nearby tests), it will also allow to test inliing against binaries